# Goal
string gz_partition
---
# Result
bool success
